readme.md

```diff
@@ -56,6 +56,10 @@
 
 1. Import the individual feature classes contained within the feature datasets
 1. Import everything outside the feature dataset
+1. Need to transfer the UDEQ.UICADMIN.UICSCHEMATRONERRORCODES by hand
+1. Set the bridging / intermediate table for UICAOR to UICArtPen permissions for all users to Select, Insert, Update and Delete
```

Comment:
I think the execute privilege removes the need for this but i could be wrong

Reply:
Candace was pretty adamant about it.  Doesn't hurt anything that I can think of.